tests/nersc_test.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+"""NERSC-related tests.
+
+:copyright: Copyright (c) 2023 RadiaSoft LLC.  All Rights Reserved.
+:license: http://www.apache.org/licenses/LICENSE-2.0.html
+"""
+
+
+def test_nersc_project(fc):
+    from sirepo import nersc
+    from pykern.pkunit import pkeq, pkok, pkre, pkfail, pkexcept
+
+    _NO_SUCH_PROJECT = "NOT_" + nersc.VALID_TEST_ACCOUNT
+
+    with pkexcept(nersc.invalid_project_msg(_NO_SUCH_PROJECT)):
+        nersc.assert_project(_NO_SUCH_PROJECT)
+
+    pkeq(
```

Comment:
Tests should dernormalize this state. You don't need to match exactly. You want to know that the logic path is followed so you pkre and partially match what you expect.

Reply:
[f5c7773](https://github.com/radiasoft/sirepo/pull/5493/commits/f5c7773f6e8928ded548737d1aacb5fb99dd8471)